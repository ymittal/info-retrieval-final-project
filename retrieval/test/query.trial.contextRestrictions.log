Dec 20, 2017 12:58:45 PM org.lemurproject.galago.core.tools.apps.BatchSearch run
INFO: RUNNING: TEXT-search : laura morelli
Dec 20, 2017 12:58:45 PM org.lemurproject.galago.core.tools.apps.BatchSearch run
INFO: Transformed Query:
#combine:w=1.0(
    #dirichlet:collectionLength=731787:maximumCount=5:nodeFrequency=16:w=0.5(
        #lengths:document:part=lengths()
        #counts:laura:part=postings.porter()
      )
    #dirichlet:collectionLength=731787:maximumCount=4:nodeFrequency=8:w=0.5(
        #lengths:document:part=lengths()
        #counts:morelli:part=postings.porter()
      )
  )

Dec 20, 2017 12:58:45 PM org.lemurproject.galago.core.tools.apps.BatchSearch run
INFO: RUNNING: LEADPARA-search : (laura morelli).(leadpara)
Dec 20, 2017 12:58:45 PM org.lemurproject.galago.core.tools.apps.BatchSearch run
INFO: Transformed Query:
#combine:w=1.0(
    #dirichlet:collectionLength=731787:maximumCount=0:nodeFrequency=0:w=0.3333333333333333(
        #lengths:document:part=lengths()
        #counts:@/(/:part=postings.porter()
      )
    #dirichlet:collectionLength=731787:maximumCount=5:nodeFrequency=16:w=0.3333333333333333(
        #lengths:document:part=lengths()
        #counts:laura:part=postings.porter()
      )
    #dirichlet:collectionLength=731787:maximumCount=4:nodeFrequency=8:w=0.3333333333333333(
        #lengths:document:part=lengths()
        #counts:morelli:part=postings.porter()
      )
  )

Dec 20, 2017 12:58:45 PM org.lemurproject.galago.core.tools.apps.BatchSearch run
INFO: RUNNING: HEADLINE,LEADPARA,TITLE-search : (laura morelli).(leadpara,headline,title)
Dec 20, 2017 12:58:45 PM org.lemurproject.galago.core.tools.apps.BatchSearch run
INFO: Transformed Query:
#combine:w=1.0(
    #dirichlet:collectionLength=731787:maximumCount=0:nodeFrequency=0:w=0.3333333333333333(
        #lengths:document:part=lengths()
        #counts:@/(/:part=postings.porter()
      )
    #dirichlet:collectionLength=731787:maximumCount=5:nodeFrequency=16:w=0.3333333333333333(
        #lengths:document:part=lengths()
        #counts:laura:part=postings.porter()
      )
    #dirichlet:collectionLength=731787:maximumCount=4:nodeFrequency=8:w=0.3333333333333333(
        #lengths:document:part=lengths()
        #counts:morelli:part=postings.porter()
      )
  )

Dec 20, 2017 12:58:45 PM org.lemurproject.galago.core.tools.apps.BatchSearch run
INFO: RUNNING: ALL.search : (laura morelli).(head,headline,leadpara,text,title)
Dec 20, 2017 12:58:45 PM org.lemurproject.galago.core.tools.apps.BatchSearch run
INFO: Transformed Query:
#combine:w=1.0(
    #dirichlet:collectionLength=731787:maximumCount=0:nodeFrequency=0:w=0.3333333333333333(
        #lengths:document:part=lengths()
        #counts:@/(/:part=postings.porter()
      )
    #dirichlet:collectionLength=731787:maximumCount=5:nodeFrequency=16:w=0.3333333333333333(
        #lengths:document:part=lengths()
        #counts:laura:part=postings.porter()
      )
    #dirichlet:collectionLength=731787:maximumCount=4:nodeFrequency=8:w=0.3333333333333333(
        #lengths:document:part=lengths()
        #counts:morelli:part=postings.porter()
      )
  )
