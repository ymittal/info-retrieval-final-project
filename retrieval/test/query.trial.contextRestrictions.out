TEXT-search Q0 B5x5CnwQZD4 1 -4.40423593 galago
